api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -34,6 +34,7 @@
  */
 public final class GlobalOpenTelemetry {
 
+  public static final Logger API_USAGE_LOGGER = Logger.getLogger("ApiUsageLogging");
```

Comment:
Can you move this to a class inside the `internal` package, maybe called `ValidationUtil` as it's logging related to validation? And we'd like to not expose it in the user API

Reply:
Let's also prefix the logger name with `io.opentelemetry`

Reply:
Done. Only a small remark: enabling tracing on `io.opentelemetry` would include these logs too (maybe we could have considered just use TRACE on a standard per-class logger